# Google back as a search engine, where ungoogled-chromium put "No Search".
# Plain URLs, no {google:...} parameters, no prefetch or location header;
# suggestions on (JD, 2026-09-28). Not the default: default-search.patch
# keeps DuckDuckGo. The data version bump makes profiles from release 1
# pick up the change once.
#
diff --git a/third_party/search_engines_data/resources/definitions/prepopulated_engines.json b/third_party/search_engines_data/resources/definitions/prepopulated_engines.json
--- a/third_party/search_engines_data/resources/definitions/prepopulated_engines.json
+++ b/third_party/search_engines_data/resources/definitions/prepopulated_engines.json
@@ -28,7 +28,7 @@
     // existing data should get a new version. Otherwise, existing data may
     // continue to be used and updates made here will not always appear.
     // Also then run tools/search_engine_choice/generate_search_engine_icons.py.
-    "kCurrentDataVersion": 211
+    "kCurrentDataVersion": 212
   },
 
   // The following engines are included in country lists and are added to the
@@ -172,15 +172,12 @@
     },
 
     "google": {
-      "name": "No Search",
-      "keyword": "nosearch",
-      "favicon_url": "about:blank",
-      "search_url": "http://{searchTerms}",
-      "new_tab_url": "about:blank",
+      "name": "Google",
+      "keyword": "google.com",
+      "favicon_url": "https://www.google.com/favicon.ico",
+      "search_url": "https://www.google.com/search?q={searchTerms}&ie={inputEncoding}",
+      "suggest_url": "https://www.google.com/complete/search?client=chrome&q={searchTerms}&ie={inputEncoding}",
       "type": "SEARCH_ENGINE_GOOGLE",
-      "preconnect_to_search_url" : "ALLOWED",
-      "prefetch_likely_navigations" : "ALLOWED",
-      "send_x_geo_header": true,
       "id": 1
     },
 
